src/PostSharp.Engineering.BuildTools/Build/Solutions/DocFxSolution.cs

```diff
@@ -0,0 +1,109 @@
+﻿using JetBrains.Annotations;
+using PostSharp.Engineering.BuildTools.Build.Model;
+using PostSharp.Engineering.BuildTools.Utilities;
+using System.Collections.Immutable;
+using System.IO;
+using System.IO.Compression;
+using System.Text.RegularExpressions;
+
+namespace PostSharp.Engineering.BuildTools.Build.Solutions;
+
+[PublicAPI]
+public class DocFxSolution : Solution
```

Comment:
This class has implementation details of _another_ repo e.g. the `docfx` directory, the name of the project, and so on. Either resolve these problems or keep the class in its original repo.